react/features/file-sharing/components/web/FileSharing.tsx

```diff
@@ -264,82 +280,103 @@ const FileSharing = () => {
-                                            src = { getFileIcon(file.fileType) } />
-                                    </div>
-                                    <div className = { classes.fileItemDetails }>
-                                        <div className = { classes.fileName }>
-                                            { file.fileName }
-                                        </div>
-                                        <div className = { classes.fileSize }>
-                                            { formatFileSize(file.fileSize) }
-                                        </div>
-                                    </div>
-                                    <div className = { `${classes.fileTimestamp} timestampVisibility` }>
-                                        <pre>
-                                            { formatTimestamp(file.timestamp) }
-                                        </pre>
-                                    </div>
-                                    <div className = { classes.buttonContainer }>
-                                        <Icon
-                                            className = { `${classes.actionIcon} actionIconVisibility` }
-                                            color = { BaseTheme.palette.icon01 }
-
-                                            // eslint-disable-next-line react/jsx-no-bind
-                                            onClick = { () => dispatch(downloadFile(file.fileId)) }
-                                            size = { 24 }
-                                            src = { IconDownload } />
-                                        { isModerator && (
-                                            <Icon
-                                                className = { `${classes.actionIcon} actionIconVisibility` }
-                                                color = { BaseTheme.palette.icon01 }
-
-                                                // eslint-disable-next-line react/jsx-no-bind
-                                                onClick = { () => dispatch(removeFile(file.fileId)) }
-                                                size = { 24 }
-                                                src = { IconTrash } />
-                                        ) }
-                                    </div>
-                                </>
-                            ) }
-                            { (file.progress ?? 100) < 100 && (
-                                <div className = { classes.progressBar }>
-                                    <div
-                                        className = { classes.progressFill }
-                                        style = {{ width: `${file.progress}%` }} />
+            {
+                sortedFiles.length > 0 && (
+                    <div className = { classes.fileList }>
+                        {
+                            sortedFiles.map(([ fileId, file ]) => (
```

Comment:
This is not correct, Array.map will pass the value as the first element, and that value itself has the ID as an attribute.